install.sh

```diff
@@ -0,0 +1,9 @@
+os=$([ $(uname | grep 'Darwin') ] && echo 'darwin' || echo 'linux');
+curl -s https://api.github.com/repos/livepeer/go-livepeer/releases/tags/0.1.7 \
```

Comment:
Should we use `latest` instead of hardcoding one specific version?

Reply:
👍

Reply:
That's definitely the ideal option.
I used a hard-coded version here because that tag doesn't seem to exist in the repo. Unless I'm just making this request improperly:
```bash
➜  ~ curl -s https://api.github.com/repos/livepeer/go-livepeer/releases/tags/latest
{
  "message": "Not Found",
  "documentation_url": "https://developer.github.com/v3/repos/releases/#get-a-release-by-tag-name"
}
```

Reply:
That's annoying... It was because we were publishing the releases as "pre-releases" - even though there is a bool field in the json payload called `pre-releases`.

I just re-published the release.  From now on, we will just not publish them as "pre-releases" anymore.

Reply:
Awesome. Updated to request the `latest` release @dob @joewest @ericxtang